source/features/show-user-top-repositories.tsx

```diff
@@ -14,13 +14,9 @@ function buildUrl(queryField: string): URL {
 	return url;
 }
 
-function init(): false | void {
-	const showcaseTitle = select('.js-pinned-items-reorder-container .text-normal');
-	if (!showcaseTitle) {
-		return false;
-	}
-
-	showcaseTitle.firstChild!.after(
+function init(): void {
+	// Show title case
+	select('.js-pinned-items-reorder-container .text-normal')?.firstChild?.after(
```

Comment:
```suggestion
	select('.js-pinned-items-reorder-container .text-normal')?.firstChild!.after(
```

You can disable the related ESLint rule **globally** if it complains.

Reply:
Should I change the 2 other places I found that i could do this as part of this pr?

Reply:
Yes please

Reply:
I made a mistake. never mind